### DISCUSSION TITLE
Categorias iniciais

### DISCUSSION THREAD

**Discussion by davidtcdeveloper** [androiddevbr/discussoes#6] [Meta]
Acho que para publicar este repositório precisamos de um conjunto claro e não muito extenso de categorias. Gostaria de propor algumas e ver com vocês quais categorias fazem sentido. Algumas que eu imagino:

🚀 Jetpack: Discussões sobre as diversas bibliotecas, recomendações e afins. Exceto Compose, que tem categoria própria.
📱 Flutter: Discussões sobre a tecnologia, dúvidas e tópicos em geral.
💻 Android Studio: Discussões sobre a IDE e as ferramentas embarcadas nela.

Já criei também a categoria Meta, para discussões como esta.

O que acham? Outras sugestões de categorias?

**Comment by walmyrcarvalho** [comment]
@davidtcdeveloper Eu curti bastante, acho que vai ser bom a gente puxar essas categorias mais gerais, depois ir agregando outras mais procuradas conforme a necessidade, por mim fechou! :)

**Reply by roquebuarquejr** [reply]
Acho que é uma boa esses 3 tópicos pra gente ir se adaptando e ver como as discussões vão rolar. 
Minha dúvida é, como expandir isso pra comunidade? Teremos alguma integração semelhante ao que temos no vagas?

**Reply by walmyrcarvalho** [reply]
@RBJUR Acho que rola de integrar algum bot pelo menos pra avisar de novas discussões por aqui, talvez seja uma!

**Reply by davidtcdeveloper** [reply]
Criei as categorias novas.
Vou dar uma olhada se tem como usar a integração do GitHub com o Slack pra notificar em caso de novas discussões.

**Reply by davidtcdeveloper** [reply]
Pelo que vi, não tem API para as discussions ainda, acho que não vai ter como integrar automaticamente com o Slack :(

https://github.com/integrations/slack/issues/1026

Acho que podemos sugerir para a galera usar a opção de subscribe para receber as notificações por e-mail. O que acham?
